torch/_torch_docs.py

```diff
@@ -1209,6 +1209,42 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.tensor_split,
+           r"""
+tensor_split(input, indices_or_sections, dim=0) -> List of Tensors
+
+Split a tensor into multiple sub-tensors, all of which are views of the
+input tensor. This function is based on :func:`numpy.array_split`.
+
+Args:
+    input (Tensor): the tensor to split
+    indices_or_sections (int or (list(int))):
+        If :attr:`indices_or_sections` is an integer, it represents the number of
```

Comment:
Maybe something like:

"... is an integer n, :attr:\`input\` is split into n sections along dimension :attr:\`dim\`. If :attr:\`input\` is divisible by `n` along dimension :attr:\`\dim\`..."

Reply:
Done